controls/cis_rhel7.yml

```diff
@@ -1686,616 +1545,669 @@ controls:
-      - var_account_disable_post_pw_expiration=30
       - accounts_set_post_pw_existing
+      - var_account_disable_post_pw_expiration=30
 
-  - id: 5.5.1.5
+  - id: 4.5.1.5
     title: Ensure all users last password change date is in the past (Automated)
     levels:
-    - l1_server
-    - l1_workstation
+      - l1_server
+      - l1_workstation
     status: automated
     rules:
       - accounts_password_last_change_is_in_past
 
-  - id: 5.5.2
-    title: Ensure system accounts are secured (Automated)
+  - id: 4.5.2.1
+    title: Ensure default group for the root account is GID 0 (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
+    rules:
+      - accounts_root_gid_zero
+
+  - id: 4.5.2.2
+    title: Ensure root user umask is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: pending
+    notes: |-
+      There is no rule to ensure umask in /root/.bash_profile and /root/.bashrc. A new rule have
+      to be created. It can be based on accounts_umask_interactive_users.
+
+  - id: 4.5.2.3
+    title: Ensure system accounts are secured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: automated
     rules:
       - no_password_auth_for_systemaccounts
       - no_shelllogin_for_systemaccounts
 
-  - id: 5.5.3
-    title: Ensure default group for the root account is GID 0 (Automated)
+  - id: 4.5.2.4
+    title: Ensure root password is set (Automated)
     levels:
-    - l1_server
-    - l1_workstation
+      - l1_server
+      - l1_workstation
     status: automated
     rules:
-      - accounts_root_gid_zero
+      - ensure_root_password_configured
 
-  - id: 5.5.4
-    title: Ensure default user shell timeout is configured (Automated)
+  - id: 4.5.3.1
+    title: Ensure nologin is not listed in /etc/shells (Automated)
     levels:
-    - l1_server
-    - l1_workstation
-    status: partial
+      - l2_server
+      - l2_workstation
+    status: pending
     notes: |-
-      The OVAL properly checks the variable but not if it is exported and readonly.
-      The Bash remediation ensures it is exported and readonly. OVAL and Ansible remediation
-      need to be incremented for CIS.
+      It is necessary to create a new rule to check and remove nologin from /etc/shells.
+      The no_tmux_in_shells rule can be used as referece.
+
+  - id: 4.5.3.2
+    title: Ensure default user shell timeout is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: automated
     rules:
       - accounts_tmout
       - var_accounts_tmout=15_min
 
-  - id: 5.5.5
+  - id: 4.5.3.3
     title: Ensure default user umask is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: automated
+    rules:
+      - accounts_umask_etc_bashrc
+      - accounts_umask_etc_login_defs
+      - accounts_umask_etc_profile
+      - accounts_umask_interactive_users
```

Comment:
I think this rule is out of the scope of this control. Maybe it could be in related_rules?

Reply:
This rule comments `umask` definition in initialization files of interactive users if they are found. This ensures the system default value is used. If there `umask` is not found, nothing is changed. However, reading again this requirement, it seems the policy is now allowing the user to change the default:
```
The user creating the directories or files has the discretion of changing the permissions via the chmod command, or choosing a different default umask by adding the umask command into a User Shell Configuration File, (.bash_profile or .bashrc), in their home directory.
```
Moved to related_rules and included a note.